src/main/java/core/basesyntax/service/impl/DataValidatorServiceImpl.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.service.impl;
+
+import static core.basesyntax.model.FruitTransaction.greatOperationPattern;
```

Comment:
Avoid using static import for 'greatOperationPattern'. Use 'FruitTransaction.greatOperationPattern()' instead.